make/autoconf/build-aux/config.guess

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 #
-# Copyright (c) 2012, 2021, Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2012, 2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
I didn't see any changes made in this file. So why change the copyright header?

Reply:
Done